packages/client/src/object/object.test.ts

```diff
@@ -75,6 +82,31 @@ describe("OsdkObject", () => {
       // it should have the prototype that we assign at hydration time
       expect(Object.keys(employee.$link.lead)).toBeDefined();
     });
+    it("loads an employee with custom client", async () => {
```

Comment:
Added another test to make sure the API url appending works as intended

Reply:
So we know the url appending works since we're hitting the right stub right?